tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -123,6 +123,10 @@ done
 
 BAZEL_FLAGS=$(str_strip "${BAZEL_FLAGS}")
 
+if [[ -z "$GIT_TAG_OVERRIDE" ]]; then
+  BAZEL_FLAGS+=" --action_env=GIT_TAG_OVERRIDE=${GIT_TAG_OVERRIDE}"
```

Comment:
lgtm. But just FYI 

--action_env=GIT_TAG_OVERRIDE

... is shorthand for ...

--action_env=GIT_TAG_OVERRIDE=${GIT_TAG_OVERRIDE}"